swagger/v1/swagger.yaml

```diff
@@ -48,35 +48,35 @@ paths:
                 date: "2023-06-11"
                 due_date: "2023-06-21"
                 city: "Windhoek"
-                user_id: 1
+              book_ids: [1, 2]
```

Comment:
![image](https://github.com/leloufadel/full-stuck-project/assets/91759364/9c4a26a8-c902-4265-a111-1d6be96560d4)

If possible, try to also add to the description or to the form of the documentation that a user can select the parameters of `book_ids`